virttest/guest_agent.py

```diff
@@ -875,3 +875,30 @@ def get_osinfo(self):
         cmd = "guest-get-osinfo"
         self.check_has_command(cmd)
         return self.cmd(cmd)
+
+    def get_memory_block_info(self):
+        """
+        Get information relating to guest memory blocks.
+        """
+        cmd = "guest-get-memory-block-info"
+        self.check_has_command(cmd)
+        return self.cmd(cmd=cmd)
+
+    def get_memory_blocks(self):
+        """
+        Get the list of the guest's memory blocks.
+        """
+        cmd = "guest-get-memory-blocks"
+        self.check_has_command(cmd)
+        return self.cmd(cmd=cmd)
+
+    def set_memory_blocks(self, mem_blocks_list):
+        """
+        Reconfigure (currently: enable/disable) state of guest memory blocks.
+        :param mem_blocks_list: a list of all memory blocks the guest
```

Comment:
From my understanding this should not be 'all' memory blocks, but only the ones you want to configure. Isn't it?

Reply:
You are right, updated.